lib/stateoscope/visualizer.rb

```diff
@@ -10,8 +10,11 @@ def parse_graph
       add_state_transitions
     end
 
-    def output(filename)
-      @viz.output(pdf: "#{filename}.pdf")
+    def output(filename, format)
+      options = {}
+      options[format] = "#{filename}.#{format}"
+      @viz.output(options)
```

Comment:
This can be boiled down to

```
@viz.output(format => "#{filename}.#{format}")
```

However, I don't like how the responsibility of generating the filename is now split across two classes. I'd rather add a `format` argument to `Stateoscope.filename_for` so we can simply use the full filename here. What do you think?

BTW I realize it was that way before, my bad 😉

Reply:
you are right. gonna change it